x-pack/plugins/cases/server/client/factory.ts

```diff
@@ -71,22 +65,14 @@ export class CasesClientFactory {
     scopedClusterClient,
     savedObjectsService,
   }: {
-    // TODO: make these required when the case connector can get a request and savedObjectsService
-    request?: KibanaRequest;
-    savedObjectsService?: SavedObjectsServiceStart;
+    request: KibanaRequest;
+    savedObjectsService: SavedObjectsServiceStart;
     scopedClusterClient: ElasticsearchClient;
   }): Promise<CasesClient> {
     if (!this.isInitialized || !this.options) {
       throw new Error('CasesClientFactory must be initialized before calling create');
     }
 
-    // TODO: remove this
```

Comment:
Removing this. It was here to not cause type errors from the case connector not having a `request` or `savedObjectsService`.